src/ramses_rf/device/hvac.py

```diff
@@ -371,6 +371,13 @@ def exhaust_flow(self) -> float | None:
 
     @property
     def exhaust_temp(self) -> float | None:
+        if Code._12A0 in self._msgs:
```

Comment:
The way I did this was to look at both `12A0` and `31DA`, and return the value in the most recent of the two - you'll see examples of this elsewhere.

Reply:
For example line 438 uses the same key for both codes. That is different here, where the key in 12A0 is just 'TEMPERATURE'. I chose to leave the 31DA return as is (and couldn't refactor/understand to use the shorthand style used in line 438 to my case).